### DISCUSSION TITLE
Maintainer wanted

### DISCUSSION THREAD

**Discussion by unseen-ninja** [cascadefox/cascade#116] [Announcements]
![maintainer](https://github.com/andreasgrafen/cascade/assets/35840154/8a9df8d3-305f-45a8-baec-fd5a5827cec8)  

It hurts me to say this, but it's for the best. As some probably have noticed: This repo got quite stale. The reason for this is that I'm not really finding much time to work on this, iron out bugs and things that got broken due to Firefox updates. Furthermore I've recently switched to another browser which already offers quite a few of the ultimate features I wanted for Cascade by default and in a way that is a lot better than what I could have done with Firefox. I'm looking to move this repository to an Org in order for potential new maintainers to have an easy transition into working on this. Again; I'm sorry, but y'all deserve better than my on-off relationship with this theme. 
\- @unseen-ninja 

---

update 23/05/26
I tried picking up maintenance of this theme early last year, continuing the work of @unseen-ninja (see above) but have admittedly been inactive fixing issues- this is probably due to not realising how annoying it is to deal with firefox's css, especially when they update selectors which leads to dead code and broken styling here. I have also been pretty busy while prioritising other projects over this as they are more enjoyable and educational for me, and I don't use this theme anymore after switching to chromium for the performance improvements...
\- @42willow

**Comment by vipexv** [comment]
What browser did you switch to?

I wanna move to Arc but haven't gotten invite for Windows beta yet :(

**Reply by unseen-ninja** [reply]
ah.. bummer. But yeah, I'm pretty much daily driving Arc myself now.

**Reply by vipexv** [reply]
That's what i was gonna guess, couldn't see a different browser other than Arc to move to but still no invite, hoping i can get it soon, hope you find a maintainer and hope you have a good day mane!

**Reply by unseen-ninja** [reply]
Thank! We'll se if someone wants to pick up this project. If not … that's okay too. There are so many amazing one-line themes out there, that user surely will find an alternative. c:

**Reply by Jamir-boop** [reply]
does Arc offer a feature to display tabs and url bar on the same row??

**Reply by unseen-ninja** [reply]
No, instead you get a sidebar with all your tabs and favourites, which can be completely hidden from view; while the URL bar is represented more in the style of a command palette. It's different from Cascade for sure, but it mimics the same layout as NVIM/VS Code/Obsidian/... and other software I use on a daily basis.

**Reply by Zabooby** [reply]
Not sure if you'd be interested, but if you still value the privacy of Firefox there are currently a few projects attempting to mimic Arc:
https://github.com/betterbrowser/arcfox
https://github.com/Naezr/ShyFox
https://github.com/KiKaraage/ArcWTF

**Comment by Techno-Simpola** [comment]
@andreasgrafen  I can maintain this, I like this theme for Firefox and want to add some small tweaks too.

**Comment by FakePickle** [comment]
@andreasgrafen So there are two of us who would like to maintain this firefox theme, I have been personally using this theme for the last year and I would love to make some improvements I have in mind.

**Comment by unseen-ninja** [comment]
@Techno-Simpola @FakePickle I've just moved Cascade over to an Org and sent both of you an invite. Imma be honest: I'm not quite sure how to best organise this now.. I was not expecting *two* people to immediately show interest.

**Reply by unseen-ninja** [reply]
Usually I'd say a chat is a nice place to figure something out tho? ¯\_(ツ)_/¯

**Reply by FakePickle** [reply]
is it also possible for a friend of mine to join in on this as well? His username is @aturtle4

**Reply by unseen-ninja** [reply]
Sure, I don't mind. The more the marrier? :D

**Reply by FakePickle** [reply]
Where do we chat regarding the same? As I would also like some input from you for the same,

**Reply by unseen-ninja** [reply]
Personally I'm mostly on Discord nowadays due to other projects as well; so I'd prefer that. But I'm open to other solutions as it mostly has to work for you *new* maintainers.

**Reply by FakePickle** [reply]
Discord sounds great!
My discord id is fakepickle11204
and @aturtle4 discord id is aturtle0184.

Also may i know your discord id?

**Reply by unseen-ninja** [reply]
Ah; neither of you are on the FirefoxCSS server. I've sent both of you a friend request then. c:
